src/connection/BaseConnectionOptions.ts

```diff
@@ -115,6 +115,11 @@ export interface BaseConnectionOptions {
      */
     readonly extra?: any;
 
+    /**
+     * The directory where ormconfig has been read from
```

Comment:
It's not clear from a comment what this option does.

Reply:
ConnectionOptions should not depend on `ormconfig.json`. 

`ormconfig.json` is just one of the connection options source.

Reply:
It exposes the [baseDirectory](https://github.com/typeorm/typeorm/blob/ceafb9e40be7702fc9f965bc62af125779f4beab/src/connection/ConnectionOptionsReader.ts#L205) so it can be used to create absolute paths to the [database files](https://github.com/elmpp/typeorm/blob/e0305a753294dd3496921c69022f4aee63950a85/src/driver/better-sqlite3/BetterSqlite3Driver.ts#L189)

I have added better comment and marked it as '@internal'